styles/claude/catppuccin.user.less

```
@@ -0,0 +1,196 @@
+    @crust: @catppuccin[@@flavor][@crust];
+    @accent: @catppuccin[@@flavor][@@accentColor];
+
+    --ctp-rosewater: @rosewater;
+    --ctp-flamingo: @flamingo;
+    --ctp-pink: @pink;
+    --ctp-mauve: @mauve;
+    --ctp-red: @red;
+    --ctp-maroon: @maroon;
+    --ctp-peach: @peach;
+    --ctp-yellow: @yellow;
+    --ctp-green: @green;
+    --ctp-teal: @teal;
+    --ctp-sky: @sky;
+    --ctp-sapphire: @sapphire;
+    --ctp-blue: @blue;
+    --ctp-lavender: @lavender;
+    --ctp-text: @text;
+    --ctp-subtext1: @subtext1;
+    --ctp-subtext0: @subtext0;
+    --ctp-overlay2: @overlay2;
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
+
+    color-scheme: if(@flavor = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    --accent-brand: #hslify(@accent) [];
+    --accent-main-000: #hslify(darken(@accent, 5%)) [];
+    --accent-main-100: #hslify(@accent) [];
+    --accent-main-200: #hslify(@accent) [];
+    --accent-main-900: #hslify(lighten(@accent, 10%)) [];
+    --accent-pro-000: #hslify(lighten(@accent, 5%)) [];
+    --accent-pro-100: #hslify(@accent) [];
+    --accent-pro-200: #hslify(@accent) [];
+    --accent-pro-900: #hslify(lighten(@accent, 10%)) [];
+    --accent-secondary-000: #hslify(@crust) [];
+    --accent-secondary-100: #hslify(@overlay0) [];
+    --accent-secondary-200: #hslify(@overlay1) [];
+    --accent-secondary-900: #hslify(@overlay2) [];
+    --bg-000: #hslify(@mantle) [];
+    --bg-100: #hslify(@base) [];
+    --bg-200: #hslify(@base) [];
+    --bg-300: #hslify(@surface0) [];
+    --bg-400: #hslify(@surface1) [];
+    --bg-500: #hslify(@surface2) [];
+    --border-000: #hslify(@base) [];
+    --border-100: #hslify(@crust) [];
+    --border-200: #hslify(@crust) [];
+    --border-300: #hslify(@overlay0) [];
+    --border-400: #hslify(@overlay1) [];
+    --border-500: #hslify(@overlay2) [];
+    --danger-000: #hslify(@maroon) [];
+    --danger-100: #hslify(@red) [];
+    --danger-200: #hslify(@red) [];
+    --danger-900: #hslify(@pink) [];
+    --oncolor-000: #hslify(darken(@mantle, 5%)) [];
+    --oncolor-100: #hslify(@mantle) [];
+    --oncolor-200: #hslify(@mantle) [];
+    --oncolor-300: #hslify(lighten(@mantle, 5%)) [];
+    --text-000: #hslify(darken(@text, 5%)) [];
+    --text-100: #hslify(@text) [];
+    --text-200: #hslify(@text) [];
+    --text-300: #hslify(@subtext0) [];
+    --text-400: #hslify(@subtext1) [];
+    --text-500: #hslify(lighten(@subtext1, 5%)) [];
+
+    /* Special overwrite for code blocks */
+    div.code-block {
+      background: @crust !important;
+
+      code {
+        color: @text !important;
+        background: @crust !important;
+      }
+    }
```

Looks like this already broke 😅, this works for me:
```suggestion
    .prismjs.code-block__code, .prismjs.code-block__code code {
      background: @crust !important;
      color: @text !important;
    }
```